### PR TITLE
Keep connections & attachments in place on host resize if possible

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: lib/features/modeling/cmd/SpaceToolHandler.js [hunk 1]
```diff
@@ -24,12 +32,20 @@ SpaceToolHandler.prototype.preExecute = function(context) {
...
+      // TODO(@barmac): replace with new Map() once we drop PhantomJS
+      oldBounds = new BoundsMap();
 
   // (1) move shapes
   this.moveShapes(movingShapes, delta);
 
-  // (2) resize shapes
+  // (2a) save old bounds of resized shapes
+  forEach(resizingShapes, function(shape) {
+    oldBounds.set(shape, getBounds(shape));
```
Comment (nikku) [inline lib/features/modeling/cmd/SpaceToolHandler.js]: Proposal: Let's not use a map here but rather a plain old object that keys by element ID (`ID` => `oldBounds`).

Comment (barmac) [reply lib/features/modeling/cmd/SpaceToolHandler.js]: Makes sense. I will do that.

Comment (barmac) [reply lib/features/modeling/cmd/SpaceToolHandler.js]: OK done.

**Comment by barmac** [comment]
@philippfromme @nikku Ready for review.

**Pull request merged**